runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnaceForemanOverlay.java

```diff
@@ -0,0 +1,73 @@
+package net.runelite.client.plugins.blastfurnace;
+
+import net.runelite.api.Client;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayMenuEntry;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+
+import javax.inject.Inject;
+import java.awt.Graphics2D;
+import java.awt.Dimension;
+
+import static net.runelite.api.MenuAction.RUNELITE_OVERLAY_CONFIG;
+import static net.runelite.client.ui.overlay.OverlayManager.OPTION_CONFIGURE;
+
+public class BlastFurnaceForemanOverlay extends Overlay
+{
+
+	private final Client client;
```

Comment:
Why are you injecting a `Client` here? It doesn't seem to be used.

Reply:
Now utilized for smithing skill lookup. Left in-tact.